### PR TITLE
Fix/design issues

### PR DESCRIPTION
**Pull request by AndriyAndriyovuch**
## Code reviewers

- [ ] @loqimean 

## Summary of issue

Change validation errors for Title field (Site Settings) #487
Change of the confirmation message #529
The language change button does not fit into the design and has a slight color overlap (it should not be so eye-catching) with the donate button. #450

## Summary of change

- fixed message in locale
- added "if" condition in model's validation
- made buttons like in mockup

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

[Запис екрану від 17.09.23 14:58:01.webm](https://github.com/ita-social-projects/ZeroWaste/assets/95449986/b567336b-0afc-499e-8341-710fa2403037)

### CODE REVIEW

File: app/assets/stylesheets/pages/main.scss [hunk 1]
```diff
@@ -73,45 +73,48 @@ a {
   }
 }
 
-.main-btns {
+.main-btn {
+
+}
+
+
```
Comment (loqimean) [inline app/assets/stylesheets/pages/main.scss:80]: анвіщо тобі ці рядки?) пустий дублюючий клас?)

File: app/models/site_setting.rb [hunk 2]
```diff
@@ -12,7 +12,8 @@ class SiteSetting < ApplicationRecord
 
   has_one_attached :favicon, dependent: :destroy
 
-  validates :title, presence: true, length: { minimum: 3, maximum: 30 }
+  validates :title, presence: true
+  validates :title, length: { minimum: 3, maximum: 30 }, if: -> { title.present? }
```
Comment (loqimean) [inline app/models/site_setting.rb:16]: а це знову ж таки навіщо?)

File: config/locales/en/en.yml [hunk 3]
```diff
@@ -486,7 +486,7 @@ en:
     category_updated: "Category was successfully updated."
     category_deleted: "Category was successfully destroyed."
     feature_flags_updated: "Features updated successfully"
-    site_setting_updated: "Setting was successfully updated"
+    site_setting_updated: "Setting were successfully updated"
```
Comment (loqimean) [inline config/locales/en/en.yml:489]: а чому множина?)

File: app/models/site_setting.rb (hunk 2 above)
Comment (AndriyAndriyovuch) [reply app/models/site_setting.rb:16]: https://github.com/ita-social-projects/ZeroWaste/issues/487

**Review by loqimean — APPROVED** [review]

**Pull request merged**